src/domains/chart/components/lib-charts/group-box-chart/kubernetes/header.tsx

```diff
@@ -4,6 +4,8 @@
 import React from "react"
 import { H5 } from "@netdata/netdata-ui"
 
-const Header = (props) => <H5 color={["white", "pure"]} wordBreak="break-all" {...props} />
+const Header = (props) => (
+  <H5 color={["white", "pure"]} wordBreak="break-all" data-testid="k8sPopover-header" {...props} />
```

Comment:
Same, maybe use `bright`?

Reply:
The problem is that the popover background color is always dark, so all the typography component should be always white.